delete-old-branches

```diff
@@ -105,6 +105,7 @@ delete_branch_or_tag() {
 main() {
     # fetch history etc
     local sha
+    git config --global --add safe.directory /github/workspace
```

Comment:
Hi @ravinaik1312 ! Thanks for this!
let's better use the env variable for the github workspace:
```suggestion
    git config --global --add safe.directory "${GITHUB_WORKSPACE}"
```